### PR TITLE
Feature/GitHub actions publisher

### PR DESCRIPTION
**Pull request by jaredweinfurtner**
From Issue:  [#22](https://github.com/carbynestack/carbynestack/issues/22)

### CODE REVIEW

File: .github/workflows/github-actions-publisher.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+name: Organizational Github Actions Publisher
```
Comment (strieflin) [inline .github/workflows/github-actions-publisher.yaml:7]: License header is missing

File: .github/workflows/github-actions-publisher.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          git add .
+          git commit -m "Updating GitHub Actions from the origanization .github repo"
+        working-directory: ./${{ matrix.target_repo }}
+      - name: Push changes
+        uses: ad-m/github-push-action@v0.6.0
+        with:
+          directory: ./${{ matrix.target_repo }}
+          repository: ${{ github.repository_owner }}/${{ matrix.target_repo }}
+          branch: master
+          github_token: ${{ secrets.CS_MINION_PAT }}
```
Comment (strieflin) [inline .github/workflows/github-actions-publisher.yaml:71]: Newline at end of file missing

File: templates/workflows/stale-issue-pr-closer.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+name: 'Close Stale Issues and PRs'
```
Comment (strieflin) [inline templates/workflows/stale-issue-pr-closer.yaml:7]: License header missing

File: templates/workflows/stale-issue-pr-closer.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
```
Comment (strieflin) [inline templates/workflows/stale-issue-pr-closer.yaml:11]: That emptyline can be removed

File: .github/workflows/github-actions-publisher.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        run: |
+          mkdir -p .github/workflows/
+          cp ../source/templates/workflows/* .github/workflows/
+        working-directory: ./${{ matrix.target_repo }}
+      - name: Publish Workflows
+        shell: bash
+        run: |
+          git config --local user.name "cs-minion"
+          git add .
+          git commit -m "Updating GitHub Actions from the origanization .github repo"
```
Comment (strieflin) [inline .github/workflows/github-actions-publisher.yaml:63]: Is it possible to have that commit signed ("-sS" option)?

**Review by strieflin — CHANGES_REQUESTED** [review]

**Comment by sbckr** [comment]
@jaredweinfurtner How about using [reusable workflows](https://github.blog/2022-02-10-using-reusable-workflows-github-actions/) instead of pushing the workflow into all other repositories?

**Comment by jaredweinfurtner** [comment]
Removing Pull Request due to alternative approach as suggested by @sbckr

**Pull request closed without merging**